web/template/layout/header.gohtml

```diff
@@ -17,6 +17,23 @@
             <li class="moj-header__navigation-item">
               <a class="moj-header__navigation-link" href="{{ sirius "/supervision" }}">Supervision</a>
             </li>
+            <li class="moj-header__navigation-item">
+              <div class="moj-search moj-search--ondark moj-search--inline">
+                <form class="form" method="get" data-module="search">
+                  <div class="govuk-form-group">
+                    <label class="govuk-label moj-search__label govuk-visually-hidden" for="search">
+                      Search
+                    </label>
+                    <input class="govuk-input moj-search__input" id="search" name="search" type="search">
+                  </div>
```

Comment:
If you set the `action` and `name` attributes, the form will automatically submit to `/search?term={term}` and you won't need the JavaScript to do it:
```suggestion
                <form class="form" method="get" action="{{ url "/search" }}" data-module="search">
                  <div class="govuk-form-group">
                    <label class="govuk-label moj-search__label govuk-visually-hidden" for="search">
                      Search
                    </label>
                    <input class="govuk-input moj-search__input" id="search" name="term" type="search">
                  </div>
```

Reply:
This will also work better when embedded, since it needs to include the prefix

Reply:
This works well however the JavaScript is needed to populate the search input on submit, so that you don't lose what you entered in the box, allowing users to edit it.

Reply:
You can keep the `window.addEventListener('load')` for that, but you shouldn't need to intercept the default submit event. I don't think it adds anything and it means the form currently doesn't work without JavaScript.